tests/test_to_json.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from resolver.to_json import ToJsonResolver
+
+
+class TestToJson(object):
+    def setup_method(self, test_method):
+        pass
+
+    def test__data_in__json_out(self):
```

Comment:
I'm not sure what we'd feed it to make it throw an error. Like, something that it doesn't have a default encoder for?

Reply:
would `None` work?

Reply:
Yeah, good idea. IIRC JSON doesn't have a `null` literal.

Reply:
Turns out [the JSON spec](https://www.json.org/json-en.html) does support `null`s. So not much to test there.

Reply:
how about try some invalid JSON?

Reply:
Still, I think you're right, we should test the failing case to make sure the right exception is returned. I'll come up with an input which will reliably fail.

Reply:
> how about try some invalid JSON?

We test the `<invalid JSON>` -> `!from_json` resolver, but `!to_json` takes any old Python object and will try to serialize it.

Reply:
ahh ok.  Forcing an invalid json is not possible since it will fail when the python parser loads the code and also our pre-commit linters will fail.  I guess we don't really need to validate the error case then.